Probesys/bileto
# new: Provide teams and agents

## Related issue(s)



https://github.com/Probesys/bileto/issues/576

## How to test manually



1. with a user with the "manage agents" permission, go to the settings > Teams & Agents
2. create a team
3. open the team and add an existing agent
4. add also an agent by his email
5. check both agents are in the team
6. check the agent added by email has been created as a user
7. remove an agent
8. check that he's no longer in the team

## Checklist
